demos/prefsFramework/index.html

```diff
@@ -61,11 +61,13 @@
         <script type="text/javascript" src="../../src/framework/preferences/js/Panels.js"></script>
         <script type="text/javascript" src="../../src/framework/preferences/js/SelfVoicingPanel.js"></script>
         <script type="text/javascript" src="../../src/framework/preferences/js/LetterSpacePanel.js"></script>
+        <script type="text/javascript" src="../../src/framework/preferences/js/LocalizationPanel.js"></script>
```

Comment:
The manual test shows the localization panel added for this demo doesn't perform a language change on the page when switching to a non-English localization option.